lib/rubocop/cop/rspec/dialect.rb

```diff
@@ -47,14 +47,14 @@ class Dialect < Cop
 
         MSG = 'Prefer `%<prefer>s` over `%<current>s`.'
 
-        def_node_matcher :rspec_method?, ALL.send_pattern
-
         def on_send(node)
-          return unless rspec_method?(node)
```

Comment:
Hmm, yeah it's definitely possible and should be done

Reply:
@pirj did you mean to define them under particular cops or in `NodePattern` module?

Reply:
Keeping it in `NodePattern` is just fine.

Reply:
Well, it looks like `rspec_method?` is used only in one place. I've implemented couple of simple methods for `Language` module for embedding keywords into block and send patterns (same as we have in `Selectors`), so used them for now.
And putted back `hook?` matcher because it's used in several places.